docs-ref-conceptual/includes/query-azure-cli-examples.md

```diff
@@ -0,0 +1,242 @@
+---
+ms.topic: include
+ms.date: 28/02/2022
+author: dbradish-microsoft
+ms.author: dbradish
+manager: barbkess
+ms.custom: devx-track-azurecli
+---
+This section contains examples of JMESPath queries for different Azure resources.
+
+### Query examples for Azure accounts
+
+This section shows example queries for storage accounts.
+
+* This examples returns the tenant ID and subscription ID of the Azure account and subscription you are using.
```

Comment:
The suggested format for bullet lists is `-` instead of `*`.
```suggestion
- This examples returns the tenant ID and subscription ID of the Azure account and subscription you are using.
```